### PR TITLE
rework about page

### PR DESCRIPTION
**Pull request by revengemiroz**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

## Showcase

Please attach a picture or a video showing what this change does.

### CODE REVIEW

File: apps/web/src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -57,9 +57,10 @@ const Footer = () => {
         <div>
           <p className='mb-6 font-mono text-2xl font-bold'>{t('explore')}</p>
           <div className='flex flex-col gap-5 text-silver'>
+            <Link href='/about'>{t('about')}</Link>
             <Link href='/search'>{t('search')}</Link>
-            <Link href='/rankings'>{t('rankings')}</Link>
-            <Link href='/privacy'>{t('privacy-policy')}</Link>
+
+            <Link href='/privacy-policy'>{t('privacy-policy')}</Link>
```
Comment (ivanms1) [inline apps/web/src/components/Footer/Footer.tsx]: this link is incorrect

File: apps/web/src/pages/About/Member/Member.tsx [hunk 2]
```diff
@@ -1,33 +1,43 @@
-import Image from 'next/future/image';
+import Image from 'next/image';
```
Comment (ivanms1) [inline apps/web/src/pages/About/Member/Member.tsx]: why did you remove future image?
bring it back and remove the wrapping div

File: apps/web/src/pages/About/Member/Member.tsx [hunk 3]
```diff
@@ -1,33 +1,43 @@
...
+        rel='noreferrer'
+        className='group relative flex w-[250px] cursor-pointer flex-col items-center justify-center gap-4 rounded-md py-8 transition-all duration-150 ease-linear hover:scale-105 hover:bg-grey-dark'
+      >
+        <div className='relative h-[100px] w-[100px] overflow-hidden rounded-full'>
+          <Image
+            src={member.avatar_url}
+            alt='github-profile'
+            layout='fill'
+            objectFit='cover'
+          />
```
Comment (ivanms1) [inline apps/web/src/pages/About/Member/Member.tsx]: remove the wrapping div and give static height and width when using future/image

File: packages/ui/package.json [hunk 4]
```diff
@@ -37,6 +37,7 @@
     "preset": "scripts/jest/node"
   },
   "dependencies": {
+    "@floating-ui/react": "^0.24.6",
     "@floating-ui/react-dom-interactions": "^0.13.3",
```
Comment (ivanms1) [inline packages/ui/package.json]: please remove this one if not used, if it's used try to replace it with the library you installed

**Review by ivanms1 — CHANGES_REQUESTED** [review]

File: apps/web/src/pages/About/About.tsx [hunk 5]
```diff
@@ -36,13 +36,13 @@ const About = ({ members }: AboutProps) => {
...
       <div>
-        <div className='flex flex-wrap justify-between gap-10 after:flex-auto after:content-[""] max-lg:justify-center'>
+        <div className='m-x-auto flex max-w-[1100px] flex-row flex-wrap  justify-center gap-2  '>
           {members.map((member) => (
             <Member key={member?.id} member={member} />
           ))}
         </div>
       </div>
-      <NextSeo title={t('seo-title')} />
+      <NextSeo title={t('about')} />
```
Comment (ivanms1) [inline apps/web/src/pages/About/About.tsx]: could please add a description and logo? 🙏 
you can copy it from the home page

File: apps/web/src/pages/About/Member/Member.tsx (hunk 3 above)
Comment (revengemiroz) [reply apps/web/src/pages/About/Member/Member.tsx]: done

File: apps/web/src/pages/About/Member/Member.tsx (hunk 2 above)
Comment (revengemiroz) [reply apps/web/src/pages/About/Member/Member.tsx]: done

File: apps/web/src/pages/About/About.tsx (hunk 5 above)
Comment (revengemiroz) [reply apps/web/src/pages/About/About.tsx]: done

File: packages/ui/package.json (hunk 4 above)
Comment (revengemiroz) [reply packages/ui/package.json]: done

**Review by ivanms1 — APPROVED** [review]

**Pull request merged**